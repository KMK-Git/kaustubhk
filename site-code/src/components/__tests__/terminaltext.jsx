import React from 'react';
import renderer from 'react-test-renderer';
import TerminalText from '../terminaltext';

const { act } = renderer;

describe('TerminalPage', () => {
  it('renders correctly', async () => {
    let tree;
    act(() => {
      tree = renderer
        .create(
          <TerminalText isFolder>
            &#39;Design Applications&#39;
            &#39;Secure Applications&#39;
            &#39;Develop Applications&#39;
            &#39;Save Money&#39;
            &#39;Develop Automation&#39;
            &#39;Cloud Native&#39;
          </TerminalText>,
        );
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
