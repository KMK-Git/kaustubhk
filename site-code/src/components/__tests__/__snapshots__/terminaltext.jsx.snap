// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TerminalPage renders correctly 1`] = `
<div
  style={
    {
      "color": "#006fb8",
    }
  }
>
  'Design Applications' 'Secure Applications' 'Develop Applications' 'Save Money' 'Develop Automation' 'Cloud Native'
</div>
`;
